airflow/providers/amazon/provider.yaml

```diff
@@ -23,14 +23,6 @@ description: |
 
 suspended: false
 versions:
-  - 8.7.1
```

Comment:
Just noticed this, will fix this when I do the second revision with additional unit tests.